### PR TITLE
feat(openrtb): Add allowed multiple size support for banner.

### PR DESCRIPTION
**Pull request by Fan-Xu**
Support format field for banner object. Format in banner is array of format objects representing the banner sizes permitted.

### CODE REVIEW

File: openrtb/banner.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import "errors"
 type Banner struct {
 	Width             int         `json:"w,omitempty"`
 	Height            int         `json:"h,omitempty"`
+	Format            []Format    `json:"format,omitempty"`
```
Comment (xiaopeng0216) [inline openrtb/banner.go]: since the format is for v2.5, could you add some comments on it.

**Comment by xiaopeng0216** [comment]
one nitpick, otherwise, LGTM

**Comment by Fan-Xu** [comment]
LGTM

**Comment by Fan-Xu** [comment]
lgtm

**Pull request merged**